rest-api-spec/src/main/resources/rest-api-spec/api/reindex.rethrottle.json

```diff
@@ -0,0 +1,24 @@
+{
+  "reindex.rethrottle": {
+    "documentation": "https://www.elastic.co/guide/en/elasticsearch/plugins/master/plugins-reindex.html",
+    "methods": ["POST"],
+    "url": {
+      "path": "/_reindex/{task_id}/_rethrottle",
+      "paths": ["/_reindex/{task_id}/_rethrottle", "/_update_by_query/{task_id}/_rethrottle"],
+      "parts": {
+        "task_id": {
+          "type": "string",
+          "description": "The task id to rethrottle"
+        }
+      },
+      "params": {
+        "requests_per_second": {
+          "type": "float",
+          "default": 0,
```

Comment:
Does this then mean that doing `curl -XPOST 'localhost:9200/_reindex/1234/_rethrottle?requests_per_second'` will set the task to be unlimited requests per sec?